test/functional/dpnsClient.spec.js

```diff
@@ -48,23 +48,22 @@ async function registerIdentity(dpp, dashCoreApi, dapiClient, options = {}) {
   const publicKeyHash = PublicKey.fromBuffer(Buffer.from(pubKeyBase, 'base64'))
     ._getID();
 
-  await dashCoreApi.generate(500);
+  await dashCoreApi.generateToAddress(500, addressString);
   await dashCoreApi.sendToAddress(addressString, 10);
-  await dashCoreApi.generate(10);
```

Comment:
Was this supposed to be replaced with a `generateToAddress` like the other places `generate` was removed?

Reply:
@thephez We did, I don't know why you're still seeing it.

Reply:
@thephez Yeah, we've switched to `0.15` version of Dash Core in our tests.

Reply:
So this line should be added here, right?
`  await dashCoreApi.generateToAddress(10, addressString);`